app/jobs/delete_user.rb

```diff
@@ -6,7 +6,10 @@ def initialize(user)
   end
 
   def perform
-    profile = User.find(user["id"])
+    profile = User.where(id: user["id"]).first
```

Comment:
Could we use `User.find_by(id: user["id"])` instead?

Reply:
thanks. updated.